packages/terra-date-picker/src/DatePicker.jsx

```diff
@@ -122,8 +122,8 @@ const defaultProps = {
   inputAttributes: undefined,
   isIncomplete: false,
   isInvalid: false,
-  maxDate: undefined,
-  minDate: undefined,
+  maxDate: '2100-12-31',
+  minDate: '1900-01-01',
```

Comment:
**Nit** : since we are adding default date for max and minDate I think we might not require these checks which were there to check if max and minDate are `undefined` : 

1.  [year_dropdown_options.jsx](https://github.com/cerner/terra-framework/blob/8bfdf4bd06564f2b59d3abeb52888b4c05ad5f98/packages/terra-date-picker/src/react-datepicker/year_dropdown_options.jsx#L84-L85) and [year_dropdown.jsx](https://github.com/cerner/terra-framework/blob/8bfdf4bd06564f2b59d3abeb52888b4c05ad5f98/packages/terra-date-picker/src/react-datepicker/year_dropdown.jsx#L79-L80)
```jsx
    const minYear = this.props.minDate ? this.props.minDate.year() : null
    const maxYear = this.props.maxDate ? this.props.maxDate.year() : null
```
2. [Index.jsx](https://github.com/cerner/terra-framework/blob/8bfdf4bd06564f2b59d3abeb52888b4c05ad5f98/packages/terra-date-picker/src/react-datepicker/index.jsx#L547)
```jsx
 const isDateRangePresent = ((typeof this.props.minDate !== 'undefined') && (typeof this.props.maxDate !== 'undefined'))
```

Reply:
@supreethmr Good call! I set the defaults for minDate and maxDate in Index.jsx and removed those checks. https://github.com/cerner/terra-framework/pull/1132/commits/f65579bd2d2d938b6b5c8c9386b763de7dca98f6